<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::ObjectInspectorWidget</class>
 <widget class="QWidget" name="GammaRay::ObjectInspectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget_1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="objectSearchLine"/>
       </item>
       <item>
        <widget class="GammaRay::ObjectsFavoriteView" name="favoritesTreeView"/>
       </item>
       <item>
        <widget class="GammaRay::DeferredTreeView" name="objectTreeView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="GammaRay::PropertyWidget" name="objectPropertyWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaRay::PropertyWidget</class>
   <extends>QWidget</extends>
   <header>propertywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GammaRay::DeferredTreeView</class>
   <extends>QTreeView</extends>
   <header>deferredtreeview.h</header>
  </customwidget>
  <customwidget>
   <class>GammaRay::ObjectsFavoriteView</class>
   <extends>QTreeView</extends>
   <header>favoritesitemview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
